visitors/http_rest/spec_visitor.go

```diff
@@ -486,13 +491,19 @@ func extendContext(cin Context, node interface{}) {
 				named = false
 			} else if x := meta.GetMultipart(); x != nil {
 				ctx.setValueType(BODY)
+				unknown := pb.HTTPBody_UNKNOWN.String()
+				contentType = &unknown
```

Comment:
At some point, we'll want to do something smarter with multipart encodings, like track the different parts.  This is fine for now though.

Reply:
Multipart metadata don't seem to have content types.